MANIFEST.in

```diff
@@ -1,3 +1,5 @@
 include README.rst
 include LICENSE
 include tox.ini
+graft docs
+graft blessings/tests
```

Comment:
There is no `tests` folder. The `tests.py` module is already included with the distro.